build.gradle.kts

```diff
@@ -120,3 +126,9 @@ tasks {
         }
     }
 }
+
+buildConfig {
```

Comment:
I can't see it from here, but I just want to make sure this generated constant is `internal`.

Reply:
The generated`BuildConfig.kt` is:

```
package build.buf.gradle

import kotlin.String

internal const val DEFAULT_BUF_VERSION: String = "1.35.0"
```